extra/1-radio-stations.js

```diff
@@ -26,6 +34,27 @@
  */
 // `getStations` goes here
 
+function getStations(){
+  const frequencies = getAllFrequencies();
+let isRadioStation = [];
```

Comment:
the fact that you call this array the same name as a function causes an error as well, as it means that in this function, isRadioStation is an array not a funtion. It's to do with scoping and other complex stuff. maybe call the array validRadioStations or something